src/clj/harja/kyselyt/yllapitokohteet.sql

```diff
@@ -804,6 +804,23 @@ WHERE yllapitokohde = :yllapitokohde
            FROM yllapitokohde
            WHERE id = :yllapitokohde) = :urakka;
 
+-- name: tallenna-yllapitokohteen-kustannukset-yhaid!
```

Comment:
Postgres sallii from lauseen käytön updatessa. Joten mä ehkä muuttaisin tämän hauin tällaiseen muotoon:
update yllapitokohteen_kustannukset 
SET kaasuindeksi = :kaasuindeksi -- ja tähän ne muutkin päivitettävät arvot
FROM yllapitokohde
WHERE
yllapitokohteen_kustannukset.yllapitokohde = yllapitokohde.id
AND yllapitokohde.urakka = :urakka-id AND yllapitokohde.yhaid = :yhaid;

Reply:
Kiitos, tämäpä oli kiintoisa. Muutin noin ja lisäsin kyselylle varmuuden vuoksi testin.